src/renderer/components/pool/PoolDetails.tsx

```diff
@@ -62,7 +65,7 @@ export const PoolDetails: React.FC<Props> = ({
           />
         </A.Col>
         <A.Col xs={24} md={16}>
-          <PoolChart isLoading={isLoading} />
+          <ChartView isLoading={isLoading} priceRatio={priceRatio} />
```

Comment:
Why do we need `isLoading` here? If I'm not wrong, data for `ChartView` depends on `getSwapHistory$` and `getDepthHistory`, which are handled by `ChartView` itself. These both streams are including a `pending` state.